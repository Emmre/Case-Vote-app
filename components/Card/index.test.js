import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import Home from "pages";

describe("Card Component", () => {
  it("Should be rendering", () => {
    const { getByTestId } = render(<Home />);
    expect(getByTestId("home-page")).toBeInTheDocument();
  });
  it("Should be snapshot", () => {
    const { getByTestId } = render(<Home />);
    expect(getByTestId("home-page")).toMatchSnapshot();
  });
});
