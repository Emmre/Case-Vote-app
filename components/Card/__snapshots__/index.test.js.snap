// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component Should be snapshot 1`] = `
<div
  class="container content"
  data-testid="home-page"
>
  <div
    class="card-content"
  />
</div>
`;
